share/instruments/instruments.xml

```diff
@@ -7279,6 +7279,125 @@
                         <controller ctrl="0" value="1"/> <!--Bank MSB-->
                         <program value="0"/> <!--Standard Kit-->
                   </Channel>
+            </Instrument>
+            <Instrument id="drum-kit-5">
+                  <family>drums</family>
+                  <!--Created as a more conventional version of drumset.-->
+                  <trackName>5-Piece Drum Kit</trackName>
+                  <longName>Drum Kit</longName>
+                  <shortName>D. Set</shortName>
+                  <description>5 piece drum kit.</description>
```

Comment:
5-piece (with hyphen)? Or maybe "Drum kit with..." and list the (physical) drums included in the kit.